waterloop-site/src/pages/Recruitment/Recruitment.tsx

```diff
@@ -43,8 +48,9 @@ const Recruitment: React.FC = () => (
         <FlexContainer>
           <h3>Our new member recruitment is now <b>closed</b> for the Winter term.</h3>
```

Comment:
Can we change this to:

Our new member recruitment for Spring has not opened yet.